src/Resources/config/routing.yaml

```diff
@@ -1,10 +1,24 @@
 sylius_refund_complete_refund_payment:
-  path: /admin/orders/{orderNumber}/refund-payments/{id}/complete
-  methods: [POST]
-  defaults:
-    _controller: PayPlug\SyliusPayPlugPlugin\Action\Admin\CompleteRefundPaymentAction
+    path: /admin/orders/{orderNumber}/refund-payments/{id}/complete
+    methods: [POST]
+    defaults:
+        _controller: PayPlug\SyliusPayPlugPlugin\Action\Admin\CompleteRefundPaymentAction
+    prefix: /admin
```

Comment:
prefix not needed as it's already in path. 

However, now sylius has a parameter for the /admin/ part, we should use it